### ISSUE TITLE
pip install wdb-lite is broken

### ISSUE THREAD

**Issue by wong2** [Kozea/wdb#74]
when running `pip install wdb-lite` to install wdb-lite, I got this error:

```
Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
    IOError: [Errno 2] No such file or directory: '/private/var/folders/6b/6nfn6_l124jblt0rmkls8l7w0000gn/T/pip-build-yWFgaB/wdb-lite/setup.py'
```

seems when you run `python setup-lite.py sdist`, the package will contain `setup-lite.py` instead of `setup.py`, so it brokes when install the package

**Comment by paradoxxxzero** [comment]
Okay fixed with 3.0.4

**Comment by wong2** [comment]
great, thanks
